src/agent/rustjail/src/container.rs

```diff
@@ -545,6 +547,8 @@ fn do_init_child(cwfd: RawFd) -> Result<()> {
     let uid = Uid::from_raw(guser.uid);
     let gid = Gid::from_raw(guser.gid);
 
+    set_stdio_permissions(guser.uid)?;
```

Comment:
This assumes that uid exists in the guest. But IMO we should check and create it before using direct, right?

Reply:
If the uid exists, would it be set as 0 by default? I added a check for root user, if the user id is '0', then it would skip the stdio permission set.

Reply:
I was wrong. No need to check it. Kernel doesn't care about `/etc/passwd` at all.